src/PostRector/Rector/UseAddingPostRector.php

```diff
@@ -68,6 +70,11 @@ public function beforeTraverse(array $nodes): array
         );
     }
 
+    public function enterNode(Node $node): int
```

Comment:
All the work for this post rector is done in the `beforeTraverse` function, so there is no need to actually traverse the nodes

Reply:
Please add comment for future reference to avoid user see and just try using STOP_TRAVERSAL blindly on rules, something like:

```
/**
 * using STOP_TRAVERSAL usually dangerous as it hit and stop
 * but since PostRector is using direct new NodeTraverser() and traverse() only on single visitor per execution
 * using stop traversal here is safe, ref https://github.com/rectorphp/rector-src/blob/fc1e742fa4d9861ccdc5933f3b53613b8223438d/src/PostRector/Application/PostFileProcessor.php#L59-L61
 */
```